MH4U Dex/ItemSourceTableViewCell.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ItemSourceTableViewCell.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/19/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "ItemSourceTableViewCell.h"
+
+#import "Area.h"
+#import "AreaDrop.h"
+#import "Monster.h"
+#import "MonsterDrop.h"
+
+@interface ItemSourceTableViewCell ()
+
+@property (nonatomic, weak) IBOutlet UILabel *methodLabel;
+@property (nonatomic, weak) IBOutlet UILabel *sourceLabel;
+@property (nonatomic, weak) IBOutlet UILabel *quantityLabel;
+@property (nonatomic, weak) IBOutlet UILabel *percentLabel;
+
+@end
+
+@implementation ItemSourceTableViewCell
+
+- (void)displayContentsWithDrop:(Drop *)drop
+{
+    if ([drop isKindOfClass:[MonsterDrop class]]) {
+        MonsterDrop *monsterDrop = (MonsterDrop *)drop;
+        self.methodLabel.text = monsterDrop.method;
+        self.sourceLabel.text = monsterDrop.monsterSource.name;
+        
+    } else if ([drop isKindOfClass:[AreaDrop class]]) {
+        AreaDrop *areaDrop = (AreaDrop *)drop;
+        self.methodLabel.text = areaDrop.method;
```

Comment:
You could move the `methodLabel` setting out of the conditional, since it's the same for both Drop types:

```objc
self.methodLabel.text = drop.method;
```